crates/bevy_ecs/src/system/mod.rs

```diff
@@ -839,4 +839,82 @@ mod tests {
             );
         }
     }
+
+    #[test]
+    fn update_archetype_component_access_works() {
+        use std::collections::HashSet;
+
+        fn a_not_b_system(query: Query<&A, Without<B>>) {
+            for a in query.iter() {
+                println!("{:?}", a);
```

Comment:
Alrighty one more baby comment then I think we should merge this: I don't think we should use println in tests (even though it wont actually print anything). Probably better to just do nothing here (or if we have a good reason to force "observation" here, maybe mutate A)?

Reply:
This will actually print things to the console when you run the test individually :)